src/codec.js

```diff
@@ -1,8 +1,8 @@
 var map = require('lodash.map')
 var filter = require('lodash.filter')
-// var log = console.log
 var convert = require('./convert')
 var protocols = require('./protocols')
+var varint = require('varint')
```

Comment:
doesn't go-multiaddr have varints for a long time? -> https://github.com/jbenet/go-multiaddr/blob/master/codec.go#L75